crates/matrix-sdk/src/sliding_sync/cache.rs

```diff
@@ -88,10 +88,7 @@ pub(super) async fn store_sliding_sync_state(
     storage
         .set_custom_value(
             instance_storage_key.as_bytes(),
-            serde_json::to_vec(&FrozenSlidingSync::new(
-                position,
-                &*sliding_sync.inner.rooms.read().await,
-            ))?,
+            serde_json::to_vec(&FrozenSlidingSync::new(&*sliding_sync.inner.rooms.read().await))?,
```

Comment:
Oh nou, this frozen thing still exists :smiling_face_with_tear:

Reply:
Yeah but its days are counted.